### PR TITLE
Rename project to Korifi

### PR DESCRIPTION
**Pull request by Birdrock**
## Is there a related GitHub Issue?
#931 

## What is this change about?
Project is being renamed to Korifi

## Does this PR introduce a breaking change?
There should be no breaking code changes.

## Acceptance Steps
Allow automation to test as usual.

## Tag your pair, your PM, and/or team
@gnovv @gcapizzi

### CODE REVIEW

**Review by gnovv — APPROVED** [review]
pairing w/ @Birdrock

File: docs/architecture-decisions/0010-merge-cf-k8s-repositories.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ Accepted
...
+The `cf-k8s-api` and `korifi` repositories were originally created separately to create a deliberate interface between "client" and "server" code. The intention was to motivate developers and users to reason about, change, and install the pieces separately. Instead, it has become difficult to decide which repository should house helper code and dependencies. Recently, we started working on a new `cf-on-k8s-integration` repository to pull the two components together and provide a home for both installation instructions and CI configuration for testing. As we worked on initialising this new repository, it became clear that different types of users wanted different things from it:
```
Comment (tcdowney) [inline docs/architecture-decisions/0010-merge-cf-k8s-repositories.md:11]: I'm not sure if we should retroactively update these to change cf-k8s-controllers to korifi.

File: controllers/controllers/workloads/integration/suite_integration_test.go [hunk 2]
```diff
@@ -110,7 +110,7 @@ var _ = BeforeSuite(func() {
 			MemoryMB:           500,
 			DefaultDiskQuotaMB: 512,
 		},
-		CFK8sControllerNamespace: "cf-k8s-controllers-system",
+		CFK8sControllerNamespace: "korifi-system",
```
Comment (tcdowney) [inline controllers/controllers/workloads/integration/suite_integration_test.go:113]: What about the stuff in `cf-k8s-api-`? Should that be korifi as well? TBH I think we should consider consolidating both of these into a single namespace, but that's probably a different story.

File: docs/architecture-decisions/0010-merge-cf-k8s-repositories.md (hunk 1 above)
Comment (Birdrock) [reply docs/architecture-decisions/0010-merge-cf-k8s-repositories.md:11]: Global search and replace :laughing: - will revert these changes.

File: controllers/controllers/workloads/integration/suite_integration_test.go (hunk 2 above)
Comment (Birdrock) [reply controllers/controllers/workloads/integration/suite_integration_test.go:113]: Something I noticed as well.

**Comment by Birdrock** [comment]
Starting fresh to avoid too much tinkering.

**Pull request closed without merging**